docs/tutorialkit.dev/src/content/docs/guides/creating-content.mdx

```diff
@@ -144,3 +158,75 @@ src/templates
     │   # Overrides "index.js" from "shared-template"
     └── index.js
 ```
+
+## Editor File Visibility
+
+Editor's files are resolved in three steps. Each step overrides previous one:
+
+1. Display files matching `template.visibleFiles` (lowest priority)
+2. Display files from `_files` directory
+3. When solution is revealed, display files from `_solution` directory. (highest priority)
+
+<Tabs syncKey="file-visibilty">
+  <TabItem label="Initially">
+
+```markdown ins=/.{24}├── (first.js)/ ins=/└── (second.js)/ ins=/third.js/
```

Comment:
This patterns might look a bit weird. It's matching the words on specific rows. I don't think expressive-code supports syntax like `ins={24, "first.js"}` to match `first.js` on line 24 only.

![image](https://github.com/user-attachments/assets/ec4b2852-c741-41f4-bcc8-52033c35fe86)

Reply:
Missing `_files/src` here 😱